contracts/dapis/DapiServer.sol

```diff
@@ -996,6 +970,32 @@ contract DapiServer is
         }
     }
 
+    /// @notice Called privately to aggregate the Beacons and return the result
+    /// @dev Tha aggregation of Beacons may have a different value than the
+    /// respective Beacon set, e.g., because the Beacon set has been updated
+    /// using signed data
```

Comment:
Can beacon sets still be updated using signed data?

Reply:
nvm, found [this](https://github.com/api3dao/airnode-protocol-v1/pull/105/files#diff-47e178e1efdc14225ddb1d731a2e39c7b341830804023013586a1893265ef2b6L974-L976)